server/tests/article.test.js

```diff
@@ -15,24 +15,24 @@ const user = {
 };
 
 const article = {
-  title: faker.lorem.sentence(),
-  body: faker.lorem.paragraph(),
-  description: faker.lorem.sentences()
```

Comment:
Faker library is causing irregularity on the test case, so i decided to replace it with plain text.